packages/schema/breadboard.schema.json

```diff
@@ -6,7 +6,7 @@
   "$defs": {
     "identifier": {
       "type": "string",
-      "pattern": "^[a-zA-Z_][a-zA-Z0-9_-]*$"
+      "pattern": "^(\\$error)|[a-zA-Z_][a-zA-Z0-9_-]*$"
```

Comment:
Still not sure this is right, because the "identifier" def is used in a bunch of places here, like node IDs. It shouldn't be valid to have a node ID called `$error`. So I think we need to factor out separate $defs, one for node IDs, one for port IDs (and one for graph IDs?).

Reply:
can do. and it would be neater than in the regex